### DISCUSSION TITLE
Welcome to aria2 Discussions!

### DISCUSSION THREAD

**Discussion by tatsuhiro-t** [aria2/aria2#2090] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by hrxn** [comment]
> Ask questions you’re wondering about.

New release when? 👀

**Comment by anasty17** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.

Socks5 proxy

**Reply by anasty17** [reply]
Add Forse start option instead of adding to queue.
Position=0 need to wait one download to finish.

**Comment by zisra** [comment]
I would love to have a m3u8/hls downloader

**Comment by xd003** [comment]
Any ETA for new release?
